src/common/utxobased/engine/makeUtxoEngine.ts

```diff
@@ -248,13 +246,15 @@ export async function makeUtxoEngine(
       const addressData = await processor.fetchAddressByScriptPubkey(
         scriptPubkey
       )
+      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
use 'ternary' instead of the '!!'. 
I actually like using '!!' but we want to make typescript happy :)